songbird/q2/_method.py

```diff
@@ -43,14 +45,16 @@ def multinomial(table: biom.Table,
     # split up training and testing
     trainX, testX, trainY, testY = split_training(
         dense_table, metadata, design,
-        training_column, num_random_test_examples
+        training_column, num_random_test_examples,
+        seed=random_seed,
     )
 
     model = MultRegression(learning_rate=learning_rate, clipnorm=clipnorm,
                            beta_mean=differential_prior,
                            batch_size=batch_size,
                            save_path=None)
     with tf.Graph().as_default(), tf.Session() as session:
+        tf.set_random_seed(random_seed)
```

Comment:
```suggestion
        if random_seed is not None:
            tf.set_random_seed(random_seed)
```
In light of the previous comment, it'll be preferable to have this instead